src/main/flight/pid.c

```diff
@@ -445,7 +445,7 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    const float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
+    const float ITermWindupPoint = ((float)pidProfile->itermWindupPointPercent - 0.001f) / 100.0f;
```

Comment:
The offset seems arbitrary and unnecessary. Also it conflicts with what @ctzsnooze done in his PR

Reply:
This is fixing divide by zero and preventing unnecesaary if else statements everywhere

Reply:
That's not the proper and clean way of avoiding division by zero, and it leads to a really big value when dividing, which is unlikely desired.

Reply:
That is how the formula is actuall. Infinite value means that you disable the feature at all.

Reply:
But multipe if statements checking if it is enabled could optimize it. I could add that.

next commit coming

Reply:
this - 0.001f just looks like some voodoo and should not be there.
it for sure is not a clean way to solve the issue and also will confuse people in the future.

Since ctznooze already is working on this part of the code i would prefer moving his pr further

Reply:
@wind0r @DieHertz 0.001 is removed now

@wind0r This is with unittests added. His PR should go on top of this one

Reply:
@wind0r  Making his changes without first modifying unittests fails to pass travis checks, but I guess you didnt read the code before your comment ;)

Reply:
@borisbstyle sure. i do not read code since i can not even code so why should i read i ;)

just to be serious. i wanted to add a extra sentence about the unit test. that those are great to have and should get merged but i thought that is so obvious that i removed it. 

with the advised to move the discussion to the other pr i also meant talking about the unit test and adding them. changes that break the unit tests of course do not get merged. i also cant remember any case where broken unit test got merged ;)

i also wrote that message because all the change here beside the unit test already where created in a pr hours befor this got created. those unit test just could get added to the other one and we just have one pr to talk about. 

i just like when helping updating prs is the way to go instead of creating a new one since in my opinion that is like "shooting cannons on sparrows"

ctznooze even allowed maintainers to push to his branch so the changes just could have been commit there without any hassle and the needed credits ;)